Assets/Scripts/Example Speech Scripts/MySpeechOutScript.cs

```diff
@@ -2,32 +2,38 @@
 using System.Collections.Generic;
 using UnityEngine;
 using System.Threading.Tasks; //note that you need to include this if you want to use Task.Delay.
+using System.Threading;
 using SpeechIO;
 
 public class MySpeechOutScript : MonoBehaviour
 {
-    // Start is called before the first frame update
     SpeechOut speechOut;
+    CancellationTokenSource source;
     void Start()
     {
+        source = new CancellationTokenSource();
+        CancellationToken token = source.Token;
         speechOut = new SpeechOut();
-        Dialog();
+        Dialog(token);
     }
 
-    public async void Dialog(){
-        await speechOut.Speak("This is a sample speech output script.");
-        await speechOut.Speak("If you hear my voice, the speech out system is functional.");
-        await speechOut.Speak("You may hear me in a different accent. If that's the case, please check your OS speech settings.");
+    public async void Dialog(CancellationToken token)
+    {
+        await speechOut.Speak("This is a sample speech output script.", token: token);
```

Comment:
here I __really__ want a back compatibility: maybe keep the Speak() without `token` ?

Reply:
The token is an optional parameter, you don't need to provide it. (For example the Dialog Scene, does not provide a token for speech)

Reply:
I see, then I won't put that in this example scripts: I intended to provide the most simple use of `speechOut.Speak()`. Maybe can you make a new example scene/script that allows cancelation?

Reply:
Alright, I created a new sample scene and script for cancelling voice output